src/crud.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable prefer-const */
+/* eslint-disable import/no-mutable-exports */
```

Comment:
[Optional]
- [ ] Please remove all linter suppression comments and fix the related issues